test/e2e/resize_nodes.go

```diff
@@ -287,27 +295,44 @@ func verifyPods(c *client.Client, ns, name string, wantName bool, replicas int)
 	return nil
 }
 
-// Blocks outgoing network traffic on 'node'. Then verifies that 'podNameToDisappear',
-// that belongs to replication controller 'rcName', really disappeared.
-// Finally, it checks that the replication controller recreates the
-// pods on another node and that now the number of replicas is equal 'replicas'.
-// At the end (even in case of errors), the network traffic is brought back to normal.
-// This function executes commands on a node so it will work only for some
-// environments.
-func performTemporaryNetworkFailure(c *client.Client, ns, rcName string, replicas int, podNameToDisappear string, node *api.Node) {
-	Logf("Getting external IP address for %s", node.Name)
-	host := ""
-	for _, a := range node.Status.Addresses {
-		if a.Type == api.NodeExternalIP {
-			host = a.Address + ":22"
-			break
-		}
+func blockNetwork(from string, to string) {
```

Comment:
Can you move the block and unblock functions into e2e/utils?
```
type netsplitHelper struct { timeout time.Time }

func (n *netsplitHelper) Block() error {}
func (n *netsplitHelper)Unblock() error {}
```

I'm alright with a follow up pr that does this, I gather many tests can use this functionality (I have one in mind already).